### PR TITLE
Product preview component

### PR DESCRIPTION
**Pull request by victorhmp**
#### What is the purpose of this pull request?
Create a Product preview component.

This PR contains commits from PRs #2 and #3 . Only the last four commits differ.

#### Screenshots or example usage

Screen Shot 2019-04-05 at 22 48 38

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  return (
+    <article
+      className={`flex justify-between bg-near-white mv3 ${
+        index % 2 === 0 ? 'mr3-m' : 'ml3-m'
+      } items-center ph6 pv5 w-40 br3`}
+    >
+      <div className="mr7">{icon}</div>
+      <div>
+        <p className="t-heading-4 serious-black">{title}</p>
+        <p className="t-body" style={{ color: '#717786' }}>
```
Comment (klzns) [inline react/Card.tsx]: Why are you setting a color here?

File: react/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+const Card: FunctionComponent<Props> = ({ icon, title, text, index }) => {
+  return (
+    <article
+      className={`flex justify-between bg-near-white mv3 ${
+        index % 2 === 0 ? 'mr3-m' : 'ml3-m'
+      } items-center ph6 pv5 w-40 br3`}
+    >
+      <div className="mr7">{icon}</div>
+      <div>
+        <p className="t-heading-4 serious-black">{title}</p>
```
Comment (klzns) [inline react/Card.tsx]: You shouldnt use `serious-black` is deprecated.

File: react/Card.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  icon: ReactNode
+  title: string
+  text: string
+  index: number
+}
+
+const Card: FunctionComponent<Props> = ({ icon, title, text, index }) => {
+  return (
+    <article
+      className={`flex justify-between bg-near-white mv3 ${
```
Comment (klzns) [inline react/Card.tsx]: You shouldnt use `near-white` is deprecated.

File: react/Hero.tsx [hunk 4]
```diff
@@ -8,20 +8,22 @@ import Brands from './images/Brands.svg'
...
+        {intl.formatMessage({ id: 'hero.resources' })}
+      </p>
       <div className="mt5">
         <Button variation="primary">
           <p className="ttu">{intl.formatMessage({ id: 'hero.cta' })}</p>
         </Button>
       </div>
       <a
-        className="tc near-white mt5"
+        className="tc near-white mt5 t-small"
```
Comment (klzns) [inline react/Hero.tsx]: near-white is deprecated

File: react/Hero.tsx [hunk 5]
```diff
@@ -8,20 +8,22 @@ import Brands from './images/Brands.svg'
 const Hero: FunctionComponent<InjectedIntlProps> = ({ intl }) => {
   return (
     <section
-      className="vh-50 mb9 pb5 bg-serious-black near-white flex flex-column items-center justify-center"
+      className="mb5 pt6 pb8 bg-serious-black near-white flex flex-column items-center justify-center"
```
Comment (klzns) [inline react/Hero.tsx]: serious-black and near-white are deprecated

File: react/Landing.tsx [hunk 6]
```diff
@@ -2,14 +2,18 @@ import React, { Fragment, FunctionComponent } from 'react'
...
+import CardGrid from './CardGrid'
+import ProductDisplay from './Product'
 
 const Landing: FunctionComponent = () => {
   const supportedLangs = ['EN', 'PT']
   return (
     <Fragment>
       <Navbar supportedLangs={supportedLangs} />
-      <main className="w-100 overflow-y-scroll">
+      <main className="w-100 bg-serious-black">
```
Comment (klzns) [inline react/Landing.tsx]: same

File: react/Product.tsx [hunk 7]
```diff
@@ -0,0 +1,25 @@
+import React, { FunctionComponent } from 'react'
+import { injectIntl, InjectedIntlProps } from 'react-intl'
+
+import productImagePath from './images/Product.svg'
+
+const ProductDisplay: FunctionComponent<InjectedIntlProps> = ({ intl }) => {
+  return (
+    <section className="w-100 flex justify-between pv10 bg-serious-black">
```
Comment (klzns) [inline react/Product.tsx]: same

File: react/Product.tsx [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+      <div className="flex flex-column justify-center w-33 pl9">
+        <p className="near-white t-small">
```
Comment (klzns) [inline react/Product.tsx]: same

File: react/Product.tsx [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+          {intl.formatMessage({ id: 'product.deliver' })}
```
Comment (klzns) [inline react/Product.tsx]: I would rather use FormattedMessage instead

File: react/Card.tsx (hunk 1 above)
Comment (victorhmp) [reply react/Card.tsx]: The color used on Figma was not in our Tachyons. I updated it to `c-muted-1`, which is the closest one.

Comment (victorhmp) [reply react/Card.tsx]: Now `c-muted-1` is this color hahaha

File: react/Card.tsx (hunk 2 above)
Comment (victorhmp) [reply react/Card.tsx]: Removed

File: react/Card.tsx (hunk 3 above)
Comment (victorhmp) [reply react/Card.tsx]: Removed

File: react/Hero.tsx (hunk 4 above)
Comment (victorhmp) [reply react/Hero.tsx]: Removed

File: react/Hero.tsx (hunk 5 above)
Comment (victorhmp) [reply react/Hero.tsx]: Removed

File: react/Landing.tsx (hunk 6 above)
Comment (victorhmp) [reply react/Landing.tsx]: Removed

File: react/Product.tsx (hunk 7 above)
Comment (victorhmp) [reply react/Product.tsx]: Removed

File: react/Product.tsx (hunk 8 above)
Comment (victorhmp) [reply react/Product.tsx]: Removed

File: react/Product.tsx (hunk 9 above)
Comment (victorhmp) [reply react/Product.tsx]: Why is that? I don't see why it would be better in this case.

**Pull request merged**